### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by tony7kh**
First homework

### CODE REVIEW

File: students/tony7kh/JS_HW1/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>Hello World!</p>
+    <script>prompt("Please enter your name")</script>
+    <script src="index.js"></script>
+</body>
+</html>
```
Comment (Temu4) [inline students/tony7kh/JS_HW1/index.html:14]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/tony7kh/JS_HW1/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+alert('Using second option for including JS')
```
Comment (Temu4) [inline students/tony7kh/JS_HW1/index.js:1]: Consider to use or single quotes `'` neither double quotes `""` across all script files

Comment (Temu4) [reply students/tony7kh/JS_HW1/index.js:1]: The same with NL

**Review by Temu4 — CHANGES_REQUESTED** [review]
Looks good! 👍  But there are a few bugs

File: students/tony7kh/JS_HW1/index.html (hunk 1 above)
Comment (tony7kh) [reply students/tony7kh/JS_HW1/index.html]: Done it

File: students/tony7kh/JS_HW1/index.js (hunk 2 above)
Comment (tony7kh) [reply students/tony7kh/JS_HW1/index.js]: Fix it

**Comment by Temu4** [comment]
@tony7kh would you pls rename folder with files related to the first task to `hw-1`?

**Comment by tony7kh** [comment]
Directory has been renamed

**Review by Temu4 — APPROVED** [review]

**Comment by Temu4** [comment]
@tony7kh could you pls make a separate PR per each task?

**Pull request merged**